index.bs

```diff
@@ -113,6 +119,68 @@ spec:html; type:dfn; for:/; text:missing value default
     <p>This is how we conceptually think about different resource types under the hood in browsers today. It may translate well
       to user-space where different types of content share similar properties.</p>
 
+    <h2 id="proposal">Proposal</h2>
+    <h3 id="fetch">Fetch</h3>
+    <ul>
+      <li>Add an <dfn id="enum-fetch-request-importance" export>Importance enum</dfn> as an <a>enumerated attribute</a> of the {{Request|Request}} class.
+        <pre class=idl>
+        enum Importance { "high", "low", "auto" };
+        </pre>
+      </li>
+
+      <li>Add a readonly <code>importance</code> attribute to the {{Request|Request}} interface of the type {Importance}.</li>
+      <li>Add a readonly <code>importance</code> attribute to the {{Request|RequestInit}} dictionary of the type {Importance}.</li>
+    </ul>
+    <p>The attribute's <i data-x="missing value default">missing value default</i> and <i
+  data-x="invalid value default">invalid value default</i> are both the <code>auto</code> importance.</p>
+    <p>The Request importance can be interpreted by the underlying browser engine to influence the prioritization of the fetch.</p>
+
+    <h3 id="html">HTML</h3>
```

Comment:
Maybe on top of this table, I think we want to actually define what the attribute mean in terms of their processing model:
* You want to define an algorithm in HTML that take and element as input and turns its `importance` attribute value (DOMString) into an importance enum value.
* Then in [update the image data](https://html.spec.whatwg.org/multipage/images.html#update-the-image-data) step 17, you want to add a step that defines the request's importance attribute to the result of that algorithm, called with the element as input.
* in [link type "preload"](https://html.spec.whatwg.org/multipage/links.html#link-type-preload) you want to add a similar step
* Same for [link type "stylesheet"](https://html.spec.whatwg.org/multipage/links.html#link-type-stylesheet) where you probably want to add to its "linked resource fetch setup steps" a step that adds the importance to the request.
* Scripts would be a bit trickier, where as part of [prepare a script](https://html.spec.whatwg.org/multipage/scripting.html#prepare-a-script) you'd want to create the importance enum value and then pass it to [Fetch a classic script](https://html.spec.whatwg.org/multipage/webappapis.html#fetch-a-classic-script) and [Fetch an external module script graph](https://html.spec.whatwg.org/multipage/webappapis.html#fetch-a-module-script-tree) where you'd set the value on the requests these algorithms create.